pytket/phir/phirgen.py

```diff
@@ -27,6 +27,8 @@
 
 logger = logging.getLogger(__name__)
 
+StrAnyDict = dict[str, Any]
```

Comment:
Doesn't seem to save that many keystrokes, perhaps best to call it something like `JsonType` to be clear of the intention if we are using an alias anyway.

Reply:
That's a better name. More than keystrokes, I wanted to signify the commonality and expectation of unified types across the file.